django_mongodb/compiler.py

```diff
@@ -86,6 +107,15 @@ def _make_result(self, entity, columns):
             result.append(obj.get(name))
         return result
 
+    def cursor_iter(self, cursor, chunk_size, columns):
+        """Generator to yield chunks from cursor."""
+        chunk = []
+        for i, row in enumerate(cursor):
+            if i % chunk_size == 0 and i > 0:
+                yield chunk
```

Comment:
possible bug, always happens. Shouldn't chunk get re-initialized here?

Reply:
the other thing, you could move line 116 up above the if and you can remove the `i > 0` check

Reply:
And, the last query. 😄 . you aren't using `len(chunk)` I think the function would be easy to red in the following way:
```python
def cursor_iter(self, cursor, chunk_size, columns):
    """Generator to yield chunks from cursor."""
    chunk = []
    for row in cursor:
        if len(chunk) == chunk_size:
            yield chunk
            chunk = []
    yield chunk
```